src/screens/CreatePost/elements/CreatePollContainer/style/CreatePollContainerBaseStyle.js

```diff
@@ -63,15 +63,15 @@ const CreatePollContainerBaseStyle = StyleSheet.create({
   },
 
   polldurationbutton: {
-    backgroundColor: colors.white,
-    color: colors.white,
+    backgroundColor: COLORS.white,
+    color: COLORS.white,
     paddingVertical: 4,
     borderRadius: 4
   },
 
   polldurationbuttontext: {
-    color: colors.white,
-    backgroundColor: colors.holytosca,
+    color: COLORS.white,
+    backgroundColor: COLORS.anon_primary,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a logical inconsistency in the `polldurationbutton` style. The `backgroundColor` and `color` properties are both set to `COLORS.white`, which would make any text on this button invisible. Please review this change.
```diff
-    backgroundColor: COLORS.white,
-    color: COLORS.white,
+    backgroundColor: COLORS.some_background_color,
+    color: COLORS.some_text_color,
```
In the `polldurationbuttontext` style, the `color` property is set to `COLORS.white` while the `backgroundColor` is set to `COLORS.anon_primary`. If `COLORS.anon_primary` is also a light color, the text might not be visible. Please ensure that there is enough contrast between the text color and the background color for readability.
```diff
-    color: COLORS.white,
-    backgroundColor: COLORS.anon_primary,
+    color: COLORS.some_text_color,
+    backgroundColor: COLORS.some_background_color,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->